src/snake/Game.java

```diff
@@ -1,5 +1,6 @@
 package snake;
 
+
 import javax.swing.*;
 
 class Game extends JFrame {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)